modules/cert-manager-acme-dns01-aws.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-aws_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Amazon Route 53.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have set up an IAM role for Amazon Route 53. For more information, see link:https://cert-manager.io/docs/configuration/acme/dns01/route53/[Route53] in the upstream cert-manager documentation.
++
+[NOTE]
+====
+For the {cert-manager-operator}, you must provide an `accessKeyID` and `secretAccessKey` to specify the credentials.
+====
+
+.Procedure
+
+. Configure the nameserver settings for the DNS-01 self check:
+
+.. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+.. Add a `spec.controllerConfig` section with the following override arguments:
++
```

Comment:
Saw @thejasn talks about the 2 args --dns01-recursive-nameservers and ...-only in https://github.com/openshift/openshift-docs/pull/56754#issuecomment-1469517451 . But for me, when I tested Route 53 in AWS and GCP clusters, I needed not set. See my verification steps in https://issues.redhat.com/browse/CM-11, I wrote there. And for dns01 other providers Azure DNS and Google Cloud DNS, I needed not set the 2 flags, either. I think they are not needed, but you can document they may be optional.

Reply:
@bergerhoffer hold off on making changes to this section for now. We are having a parallel discussion around testing this. Will update here once we have a conclusion.

Reply:
@thejasn @xingxingxia Is there any conclusion to this? Posted some followup questions in the slack thread

Reply:
Updating to say that Anjali answered in slack, so I've updated per that